wordpress-mobile/WordPress-Android
# Issue/blogging prompts attribution

This PR adds an attribution to prompts that come from Day One.

In addition I used this PR to sneak in a couple of extra fixes:
- Added a for blogging prompt card to the my site card tracking logic.
- Fixed onboarding dialog reminder caption color in dark mode.
- Replaced my faces with default avatar placeholder.

[![Image from Gyazo](https://i.gyazo.com/5076081f23ded25c9caa3d203b2d60bb.png)](https://gyazo.com/5076081f23ded25c9caa3d203b2d60bb)

[![Image from Gyazo](https://i.gyazo.com/e2f8a0fa1b1dd7bdcfe1315b570bb7a5.png)](https://gyazo.com/e2f8a0fa1b1dd7bdcfe1315b570bb7a5)

[![Image from Gyazo](https://i.gyazo.com/268d359b0da10436997551c84f45b51a.png)](https://gyazo.com/268d359b0da10436997551c84f45b51a)

[![Image from Gyazo](https://i.gyazo.com/26735b1fe0aa9d61bffaea5ee5bb74d9.png)](https://gyazo.com/26735b1fe0aa9d61bffaea5ee5bb74d9)

To test:
- Enable prompts FF and restart the app.
- Most of the prompts are from Day One, so you will probably see the Day One prompt. Make sure the attribution looks as expected.
- To hide the attribution you can manually update the value passed to the card builder [here](https://github.com/wordpress-mobile/WordPress-Android/compare/issue/blogging-prompts-attribution?expand=1#diff-e536c944784cffa9b67cd62bea37b9d8e9f34a59927b3a59edfd3495956c46a6R169) and [here](https://github.com/wordpress-mobile/WordPress-Android/compare/issue/blogging-prompts-attribution?expand=1#diff-864689428444b7268bf457395525bc503f16d51ff47dd9c4b3385cba48ac39e5R30).
- Confirm that the rest of the fixes look ok.




## Regression Notes
1. Potential unintended areas of impact
-  Attribution visibility logic is compromised in some way.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Manual testing. Existing card builder tests.

3. What automated tests I added (or what prevented me from doing so)
- View holder are pretty difficult to test, but we are making sure that the data that goes to them is what we expect.bl


PR submission checklist:
